### PR TITLE
Update Copyrights

### PR DESCRIPTION
**Pull request by vbatts**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -176,6 +176,7 @@
 
    END OF TERMS AND CONDITIONS
 
+   Copyright 2019, 2020 The Linux Foundation
```
Comment (thaJeztah) [inline LICENSE]: Looking at the list of [recommended options](https://github.com/cncf/foundation/blob/master/copyright-notices.md#copyright-notices)

Perhaps "Copyright 2019, 2020 The go-digest contributors" ?

(I personally like "authors" or "contributors", as it more clearly describes that copyright is not transferred to `<some organisation>`, but remains with the author/contributor)

**Comment by vbatts** [comment]
I agree. This is mirroring the copyright in the other OCI specs. I'm not sure of the reason for the attribution to be to Linux Foundation, rather than contributors or members or such.

-------- Original Message --------

**Comment by vbatts** [comment]
updated to copyright by "OCI Contributors"

**Comment by vbatts** [comment]
come on @opencontainers/go-digest-maintainers ...

**Comment by vbatts** [comment]
(i'm wanting to cut a 1.0.0 release #46)

**Comment by stevvooe** [comment]
Lgtm

**Comment by dmcgowan** [comment]
LGTM

**Comment by vbatts** [comment]
not sure why Stephen's LGTM didn't work? It was "Lgtm" to begin with, and didn't trigger the pullapprove. I edited to be "LGTM", but that also hasn't triggered..

**Comment by vbatts** [comment]
LGTM

**Comment by dmcgowan** [comment]
There is another PR to fix pull approve, but also pull approve doesn't work on that PR :man_facepalming:

**Comment by AkihiroSuda** [comment]
merge?

**Comment by vbatts** [comment]
@caniszczyk help?

**Comment by AkihiroSuda** [comment]
ping @caniszczyk 🙏

**Comment by philips** [comment]
LGTM (for old times sake)

**Comment by caniszczyk** [comment]
LGTM sorry, too many github notifications, message me next time :)

but yes, this falls in lines with LF guidelines
https://www.linuxfoundation.org/blog/2020/01/copyright-notices-in-open-source-software-projects/

**Pull request merged**